wcfsetup/install/files/lib/form/RegisterForm.class.php

```diff
@@ -403,6 +404,14 @@ public function save() {
 			// create fake password
 			$this->password = StringUtil::getRandomID();
 		}
+
+		$data = [
+			'saveOptions' => $saveOptions,
+			'registerVia3rdParty' => $registerVia3rdParty
+		];
+		EventHandler::getInstance()->fireAction($this, 'externalAuthentication', $data);
```

Comment:
I believe that re-using the existing `3rdParty` terminology makes this more clear:
```suggestion
		EventHandler::getInstance()->fireAction($this, 'registerVia3rdParty', $eventParameters);
```